CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 
 ## 4.0.0-RC1 / 2016-06-21
 
+* [GeneratorBundle] Use multilanguage parameter to detect multi/single language site, when generating default-site [#1086](https://github.com/Kunstmaan/KunstmaanBundlesCMS/issues/1086) ([@treeleaf](https://github.com/treeleaf))
```

Comment:
Can you remove the changes form the CHANGELOG, it will be generated automatically